kissim/cli/main.py

```diff
@@ -97,5 +100,34 @@ def main():
     )
     compare_subparser.set_defaults(func=compare_from_cli)
 
+    # Arguments and function to be called for sub-command encode
+    tree_subparser.add_argument(
+        "-i",
+        "--input",
+        type=str,
+        help="Path to a kissim kinase matrix CSV file.",
+        required=True,
+    )
+    tree_subparser.add_argument(
+        "-o",
+        "--output",
+        type=str,
+        help="Path to the output file in Newick format.",
+        required=True,
+    )
+    tree_subparser.add_argument(
+        "-c",
+        "--clustering",
+        type=str,
+        help="Clustering method.",
+        required=False,
+        choices=["ward", "complete", "weighted", "average", "centroid"],
```

Comment:
Please add the default "ward" here, otherwise _None_ will be used as clustering method, which will result in an error.
`default="ward",`